examples/gno.land/r/matijamarjanovic/config/config.gno

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"errors"
+	"std"
+)
+
+var (
+	main   std.Address // tony's main address
+	backup std.Address // backup address
+)
+
+func init() {
+	main = "g1n3s28tukh9fr37d44cucutsn22d4e8hw3666dv"
+}
+
+func Address() std.Address {
+	return main
+}
+
+func Backup() std.Address {
+	return backup
+}
+
+func SetAddress(a std.Address) error {
+	if !a.IsValid() {
+		return errors.New("config: invalid address")
```

Comment:
error messages are repeated (invalid arress and unauthorized). please make them as constant.

Reply:
Hi, pleas see fix im this commit: 15e15a2429bd67442a8e93f28eeac5ecfef1abeb

Reply:
Hi, please see fix here: ef98d67f51538ecdde5c496967a83b4d00e94673